README.md

```diff
@@ -51,3 +55,22 @@ jobs:
         env:
           GH_PAT: ${{ secrets.GH_PAT }}
 ```
+
+### Example with User Data
```

Comment:
I don't see user data?

Reply:
Good catch, fixed (linking to [OA/ec2 example] instead).

[OA/ec2 example]: https://github.com/Open-Athena/ec2/blob/94e815ac681ba5836ce07cda894d53d3dd900afd/.github/workflows/runner.yml#L83